src/coreclr/tests/external/external.csproj

```diff
@@ -26,6 +26,14 @@
     <NoWarn>$(NoWarn);NU1701</NoWarn>
   </PropertyGroup>
 
+  <!-- XUnit.Console targets netcoreapp2.1; override with rollForwardOnNoCandidateFx=2
```

Comment:
No need to hack this here. Opened https://github.com/dotnet/arcade/pull/5475

Reply:
Much nicer - thanks. I'll update this PR and wait for the arcade changes to flow through